frontend/src/components/DeliveryCard.tsx

```diff
@@ -6,18 +6,26 @@ import {useNavigate} from "react-router-dom";
 
 type CardProps = {
     delivery: DeliveryModel;
+    deleteDelivery: (id:string) => void
 }
 
 export default function DeliveryCard(props: CardProps) {
     const navigate = useNavigate();
+
+    function onDeleteClick(){
+        props.deleteDelivery(props.delivery.id)
+    }
+
     return (
         <Card variant="outlined" className="details-card">
             <small> ID: {props.delivery.id}</small>
             <p>Title: {props.delivery.title}</p>
-            <ButtonGroup sx={{display: "flex", justifyContent: "flex-end"}} variant="text"
+            <ButtonGroup sx={{display: "flex", justifyContent: "space-between"}} variant="text"
```

Comment:
Hab ich jetzt so auch noch nicht gesehen ein css code in der tsx ? damit die Button nicht direkt nebeneinander kleben ? wüsste jetzt so Spontan auch nicht wie ich das in CSS machen würde außer es mit justifyContent: "space-between" zu testen. könnt mir ja mal Morgen erklären warum so.

Reply:
Wenn du bei MUI-Komponenten ein vorgegebenes Styling überschreiben willst, geht es nur mit diesem sx={} - Inline-CSS (zumindest habe ich noch keine andere Möglichkeit gefunden, mit className kann man nur ergänzen)